### PR TITLE
Fix struct names

### PR DESCRIPTION
**Pull request by coderwander**
#### Type of change

- Documentation update

#### Description

Fix struct names

### CODE REVIEW

File: core/scc/inprocstream.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ func (e SendPanicFailure) Error() string {
 	return fmt.Sprintf("send failure %s", string(e))
 }
 
-// PeerChaincodeStream interface for stream between Peer and chaincode instance.
+// InProcStream interface for stream between Peer and chaincode instance.
```
Comment (satota2) [inline core/scc/inprocstream.go]:
```suggestion
// inProcStream interface for stream between Peer and chaincode instance.
```
For name consistency, it should start with a lowercase letter.

**Review by satota2 — CHANGES_REQUESTED** [review]
@coderwander 
I have commented on the minor modifications. Please confirm it.

File: core/scc/inprocstream.go (hunk 1 above)
Comment (coderwander) [reply core/scc/inprocstream.go]: Thanks. Modified

**Review by C0rWin — APPROVED** [review]

**Review by satota2 — APPROVED** [review]
@coderwander Thank you for your updates.

**Pull request merged**